JumpTheQueue/index.asciidoc

```diff
@@ -0,0 +1,48 @@
+= Tutorial for the Jump The Queue Application
+====
+Jump The Queue is a small application based on the devonfw framework, which you can create yourself by following our simple step-by-step tutorial. By doing so, you will learn about the app development workflow and gain insight into the design of a professional business information system.
+
+More information about Jump The Queue on https://github.com/devonfw/jump-the-queue
+====
+
+Install devonfw IDE
+[step]
+--
+installDevonfwIde(["java", "mvn", "node", "ng", "npm"])
```

Comment:
Should be restoreDevonIde because this tutorial is not on how to install devonfw-ide but on JTQ.